core/tests/protractor_utils/editor.js

```diff
@@ -377,9 +386,10 @@ var addParameterChange = function(paramName, paramValue) {
 // should be specified after the ruleName as additional arguments. For example
 // with interaction 'NumericInput' and rule 'Equals' then there is a single
 // parameter which the given answer is required to equal.
-var _selectRule = function(ruleElement, interactionId, ruleName) {
+var _selectRule = function(ruleElement, interactionId, ruleName,
+  skipParameters) {
```

Comment:
I don't understand what this parameter means, semantically. Please could you modify the docstring? (The current docstring tallks about stuff that is "specified after the ruleName", which seems incorrect with this extra argument.)

Another alternative design is to have two methods that make use of similar functions internally: _selectRule() and _selectRuleWithoutSpecifyingParameters(). Similar to how you broke up setInteraction().

Also -- do other e2e tests still run, with this change? I'm actually a bit surprised that this doesn't break stuff.